tests/test_move.py

```diff
@@ -151,6 +151,14 @@ def test_move_file_read_only_mem_dest(self):
                 dst_ro.exists("target.txt"), "file should not have been copied over"
             )
 
+    def test_overwrite(self):
+        with open_fs("temp://") as src, open_fs("temp://") as dst:
+            src.writetext("file.txt", "Content")
+            dst.writetext("target.txt", "Content")
+            fs.move.move_file(src, "file.txt", dst, "target.txt")
+            self.assertFalse(src.exists("file.txt"))
+            self.assertTrue(dst.exists("target.txt"))
```

Comment:
Perhaps it's worth giving `file.txt` and `target.txt` different contents, and checking that the content of `target.txt` _has_ actually changed after the `move_file` ? :shrug:

Reply:
And it might be being a bit overly-pedantic, but I wonder if it's worth adding
```python
self.assertTrue(src.exists("file.txt"))
self.assertFalse(dst.exists("file.txt"))
self.assertFalse(src.exists("target.txt"))
self.assertTrue(dst.exists("target.txt"))
```
before the `move_file`, just to make it blindingly obvious that `src` and `dst` are in fact separate FSes? (as it looks like the current test might actually pass if `src` and `dst` were pointing to the same FS?)

Reply:
I was suggesting adding them _beofre_ the move_file, but you've added them _after_ ?

Reply:
Ah I see. Added it now.